doc/man3/RAND_add.pod

```diff
@@ -78,6 +69,54 @@ RAND_event() returns RAND_status().
 
 The other functions do not return values.
 
+=head1 NOTES
+
+The new OpenSSL DRBG has some peculiarities which need to be taken
+into account when it is selected as the default OpenSSL CSPRNG, i.e.,
+when RAND_get_rand_method(void) == RAND_OpenSSL(). This applies in
+particular to the way reseeding is done by the DRBG:
+
+ * The DRBG seeds itself automatically, pulling randomness from trusted
```

Comment:
I would prefer if you used the usual POD construct for lists. For bulleted lists, it looks like this:

```
=over 2

=item *

First item

=item *

Next item

=item *

Etc

=back
```

(From memory)